sysrib/zapi_test.go

```diff
@@ -173,6 +174,35 @@ func testRouteAdd(t *testing.T) {
 	}
 }
 
+// TODO(wenbli): Remove
+type FakeDataplane struct {
```

Comment:
is this used somewhere? does it need to be exported?

Reply:
Yes it's used in zapi_test.go. It's in the test file so I think it's fine that it's exported for now (has always been that way).